modules/src/context_mock.rs

```diff
@@ -40,32 +40,37 @@ impl MockChainContext {
         }
     }
 
-    /// Used for testing
     pub fn validate(&self) -> Result<(), Box<dyn Error>> {
-        // check that the number of entries is not higher than max_size
-        if self.history.len() > self.max_size {
-            return Err("too many entries".to_string().into());
-        }
-
-        // check latest is properly updated with highest header height
-        let SelfHeader::Mock(lh) = self.history[self.history.len() - 1].header;
-        if lh.height() != self.latest {
-            return Err("latest height is not updated".to_string().into());
-        }
-
-        // check that all headers are in sequential order
-        for i in 1..self.history.len() {
-            let SelfHeader::Mock(ph) = self.history[i - 1].header;
-            let SelfHeader::Mock(h) = self.history[i].header;
-            if ph.height().increment() != h.height() {
-                return Err("headers in history not sequential".to_string().into());
-            }
-        }
+        // TODO
         Ok(())
+        //         // check that the number of entries is not higher than max_size
```

Comment:
Is this buggy or problematic for compilation? Or how come it is commented out?

Reply:
Just fixed it.